MIGRATING.md

```diff
@@ -0,0 +1,84 @@
+# Okta Signin Widget migration guide
+
+This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we don't make breaking changes unless the major version changes!
+
+## Migrating from 2.x to 3.x
+
+### Consolidated css files to single one
```

Comment:
suggestion: Remove "to single one" to be more concise:

### Consolidated CSS files